config/eslint.config.js

```diff
@@ -5,5 +5,13 @@ module.exports = {
     'react/require-default-props': 'warn',
     'import/no-extraneous-dependencies': false, // disabling it because all dev dependencies are pulled into the root
     'prefer-destructuring': 0,
+    'jsx-a11y/label-has-for': [
```

Comment:
Looks like we are using the wrong telus eslint config. We need `@telus/eslint-config` instead of `@telusdigital/eslint-config`. (They're separate packages) The current config is a year out of date. Maybe this should be broken off as an upgrade since this changes our overall tooling? (Also explains why there was no deprecation warning, and why the new rule wasn't found.)

Reply:
I've moved this to our backlog 👍